packages/orbit-components/src/Icon/__tests__/index.test.js

```diff
@@ -1,60 +1,24 @@
 // @flow
 import * as React from "react";
-import { mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
 
 import Accommodation from "../../icons/Accommodation";
-import { ICON_SIZES, ICON_COLORS } from "../consts";
-import defaultTheme from "../../defaultTheme";
+import { ICON_SIZES } from "../consts";
 
 describe("Icon", () => {
-  const color = ICON_COLORS.PRIMARY;
-  const component = mount(<Accommodation color={color} />);
-  const svg = component.find("svg");
-  it("should render SVG", () => {
-    expect(component.find("svg").exists()).toBe(true);
-  });
-  it("should have default sizing", () => {
-    expect(svg).toHaveStyleRule("height", defaultTheme.orbit.widthIconMedium);
-    expect(svg).toHaveStyleRule("width", defaultTheme.orbit.widthIconMedium);
-  });
-  it("should have selected color", () => {
-    expect(svg).toHaveStyleRule("color", defaultTheme.orbit.colorIconPrimary);
-  });
-  it("should have CSS properties", () => {
-    expect(svg).toHaveStyleRule("vertical-align", "middle");
-    expect(svg).toHaveStyleRule("fill", "currentColor");
-  });
-});
-
-describe("Icon with custom props", () => {
-  const dataTest = "test";
-  const size = ICON_SIZES.LARGE;
-  const customColor = "#FF0000";
-  const ariaHidden = true;
-  const ariaLabel = "Accommodation";
-  const component = mount(
-    <Accommodation
-      size={size}
-      customColor={customColor}
-      dataTest={dataTest}
-      ariaHidden={ariaHidden}
-      ariaLabel={ariaLabel}
-    />,
-  );
-  const svg = component.find("svg");
-  it("should render SVG", () => {
-    expect(component.find("svg").exists()).toBe(true);
-  });
-  it("should render attributes", () => {
-    expect(svg.render().prop("data-test")).toBe(dataTest);
-    expect(svg.render().prop("aria-hidden")).toBe(String(ariaHidden));
-    expect(svg.render().prop("aria-label")).toBe(ariaLabel);
-  });
-  it("should have default sizing", () => {
-    expect(svg).toHaveStyleRule("height", defaultTheme.orbit.widthIconLarge);
-    expect(svg).toHaveStyleRule("width", defaultTheme.orbit.widthIconLarge);
-  });
-  it("should have selected color", () => {
-    expect(svg).toHaveStyleRule("color", customColor);
+  it("should have expected DOM output", () => {
+    render(
+      <Accommodation
+        size={ICON_SIZES.LARGE}
+        customColor="#FF0000"
+        dataTest="test"
+        ariaHidden
+        ariaLabel="Accommodation"
```

Comment:
@silvenon  looks like you forgot to test `color` prop, previos test implementation had that. Or you think it's not necessary?

Reply:
Ah, I totally forgot I can just do `toHaveStyle`!